---
layout: post
title:  SpikeFlow is now published on NAR Genomics & Bioinformatics
date: 29 August 2024
inline: false
---


📣 Excited to announce our new paper, “SpikeFlow: Automated and Flexible Analysis of ChIP-Seq Data with Spike-in Control”, now online in NAR Genomics and Bioinformatics! 📣 

 📎 Find the paper at [this link](https://academic.oup.com/nargab/article/6/3/lqae118/7744948).

If you work with ChIP-seq and spike-in normalization, check it out! 

SpikeFlow is a snakemake 🐍 workflow for ChIP-Rx data analysis. It integrates several tools to simplify spike-in normalization, peak calling, and differential analysis, making it easier to compare histone modifications or transcription factors binding across biological conditions. We benchmarked it against commonly used tools and showed that proper normalization could improve the number of detected differential binding regions. 

We welcome feedback and are excited to see how SpikeFlow is used. Reach out with any questions or issues! 🖥 

- Docker image: [https://zenodo.org/records/11544955](https://zenodo.org/records/11544955)
- SpikeFlow code and documentation: [https://github.com/DavideBrex/SpikeFlow](https://github.com/DavideBrex/SpikeFlow)




{% twitter https://x.com/BrexDavide/status/1829180657807024427 %} {% twitter https://x.com/LabChiacchiera/status/1829219031586377832 %}